src/Microsoft.Health.SqlServer/Features/Schema/SchemaInformation.cs

```diff
@@ -18,5 +18,14 @@ public SchemaInformation(int minimumSupportedVersion, int maximumSupportedVersio
         public int MaximumSupportedVersion { get; }
 
         public int? Current { get; set; }
+
+        /// <summary>
+        /// Checks if the version is not inside the full schema script.
+        /// </summary>
+        /// <param name="version">The schema version.</param>
+        public static bool DoesNotContainsVersionInSchema(int version)
+        {
+            return version > 5;
+        }
```

Comment:
Coming back to this PR after thinking about it for a while - should `version > 5` be hard coded in this repo if this bool is specific to how the schema files are set up in the fhir-server repo?